### ISSUE TITLE
Implement list view of improved library layout

### ISSUE THREAD

**Issue by jacksonh** [omnivore-app/omnivore#1079]
Our new library layout has a list view you can access by tapping the list icon (beside Add Link) on the library page (`/library`). Updated designs are here: https://www.figma.com/file/ILgs6aXrPfukXOMf5Yibra/Omnivore---Deliverables?node-id=3573%3A91810

<img width="693" alt="Screen Shot 2022-08-15 at 14 19 00" src="https://user-images.githubusercontent.com/75189/184586069-b26353d4-602a-444d-9b9a-b24ffdc439b3.png">

NOTE: we are doing one more iteration of design feedback so we shouldn't start this yet. We will be adding author and siteName to this design (and the other card design).

**Comment by rupin90** [comment]
I think I can get started on this now? :D @jacksonh

**Comment by rupin90** [comment]
or may be once you merge masonry Layout

**Comment by rupin90** [comment]
Final design - 
<img width="776" alt="Screen Shot 2022-08-24 at 12 45 14 AM" src="https://user-images.githubusercontent.com/107437442/186339917-1d696b45-2760-4386-96a6-7716de38a359.png">

**Comment by jacksonh** [comment]
Yeah, i think the Masonry stuff is merged so we can go back and fix up the grid style cards to match.

**Comment by rupin90** [comment]
oopsie!!!
